app/views/event/participations/_roles_sac_cas.html.haml

```diff
@@ -3,4 +3,5 @@
 / or later. See the COPYING file at the top-level directory or at
 / https://github.com/hitobito/hitobito_sac_cas.
 
-= self_employed_label(role)
\ No newline at end of file
+- if role.kind == :leader
+  = self_employed_label(role)
```

Comment:
So this does only work if all the roles have the same kind. If there are multiple roles available which may or may not have kind `:leader` then we'd need to switch using stimulus.